pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -62,6 +66,29 @@ const (
 	// defaultIPAMDir is the default location for the checkpoint files stored by host-local ipam
 	// https://github.com/containernetworking/cni/tree/master/plugins/ipam/host-local#backends
 	defaultIPAMDir = "/var/lib/cni/networks"
+
+	zeroCIDRv6 = "::/0"
+	zeroCIDRv4 = "0.0.0.0/0"
+
+	NET_CONFIG_TEMPLATE = `{
+  "cniVersion": "0.1.0",
```

Comment:
Might as well bump this to at least v0.3.1, which has a much more useful return type.

Reply:
the concern i have is current bootstrapping tools that still use older versions. Specially those which have no intentions to use dualstack, they should keep doing what they are doing with no changes needed. Version `v0.3.1` does not add anything to dual stack context, so having two specs (one for old, one for dualstack) will just over complicate things.